org.eclipse.ice.archetypes/src/main/java/org/eclipse/ice/App.java

```diff
@@ -0,0 +1,13 @@
+package org.eclipse.ice;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
+    public static void main( String[] args )
+    {
+        System.out.println( "Hello World!" );
+    }
+}
```

Comment:
Is this file needed for the archetype? (I'm guessing no since it looks like generated content :slightly_smiling_face:)

Reply:
Yeah, I should remove that.

Reply:
And the AppTest too...